examples/stm32_f4ve/gui/images/.gitignore

```diff
@@ -0,0 +1,2 @@
+*.hpp
+*.cpp
```

Comment:
redundant due to `../.gitignore`?

Reply:
If I understand correctly, yep. Should I delete this one?

Reply:
Yes.